bus/api.go

```diff
@@ -47,6 +47,19 @@ type WalletSignRequest struct {
 	CoveredFields types.CoveredFields `json:"coveredFields"`
 }
 
+// WalletSplitRequest is the request type for the /wallet/split operation.
+type WalletSplitRequest struct {
+	Amount  types.Currency `json:"amount"`
+	Outputs int            `json:"outputs"`
+}
+
+// WalletSplitResponse is the response type for the /wallet/split endpoint.
+type WalletSplitResponse struct {
+	Transaction   types.Transaction   `json:"transaction"`
+	ToSign        []types.OutputID    `json:"toSign"`
+	CoveredFields types.CoveredFields `json:"coveredFields"`
```

Comment:
why do we need to return the covered fields?

Reply:
(strictly speaking, we don't need to return `ToSign` either, but I suppose it's convenient if you're going to be immediately calling `/wallet/sign`)

Reply:
Actually, since the `bus` wallet will (presumably) always be a hot wallet, we could have `/wallet/split` return an already-signed transaction, ready for broadcasting. 🤔

Reply:
done